tfhe/examples/utilities/params_to_file.rs

```diff
@@ -70,9 +89,30 @@ impl ParamDetails<u64> for ShortintParameterSet {
         self.polynomial_size()
     }
 
-    fn log_ciphertext_modulus(&self) -> usize {
-        assert!(self.ciphertext_modulus().is_native_modulus());
-        64
+    fn lwe_ciphertext_modulus(&self) -> ParamModulus {
+        match self {
+            Self::Standard(p) => {
+                assert!(p.ciphertext_modulus().is_native_modulus());
+                ParamModulus::Other(1u128 << self.ciphertext_modulus().associated_scalar_bits())
```

Comment:
why not do the same as KS32 below ? so that it works when we have weird moduli ?